plugins/modules/one_vm.py

```diff
@@ -1398,7 +1482,8 @@ def main():
         "labels": {"default": [], "type": "list", "elements": "str"},
         "count_labels": {"required": False, "type": "list", "elements": "str"},
         "disk_saveas": {"type": "dict"},
-        "persistent": {"default": False, "type": "bool"}
+        "persistent": {"default": False, "type": "bool"},
+        "updateconf": {"required": False, "type": "dict"},
```

Comment:
Module arguments are not required by default, and we have been trying to avoid the redundant definition:
```suggestion
        "updateconf": {"type": "dict"},
```